src/main/java/org/craftercms/studio/api/v2/service/config/ConfigurationService.java

```diff
@@ -105,4 +110,6 @@ Resource getPluginFile(String siteId, String type, String name, String filename)
      * @return configuration history
      */
     ConfigurationHistory getConfigurationHistory(String siteId, String module, String path, String environment);
+
+    void writeGlobalConfiguration(String path, InputStream content) throws ServiceLayerException;
```

Comment:
Shouldn't this also need the `write` permission?

Reply:
Interfaces do not need permission